lib/model.js

```diff
@@ -465,6 +472,18 @@ function filterable(props, subfilters) {
         data = data.split(',');
       }
 
+      if (filter_func === 'within') {
```

Comment:
Within function requires here multiple coordinate pairs (2 for square search areas or more for polygons).
When I use GET "/users?loc__within=1,1&loc__within=2,2" data variable will contain ["1,1", "2,2"] - that's the reason behind my implementation.